portfolio/src/main/webapp/inport.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+<body onload="getHistory()">
+
+<form action="/inf" method="POST">
+      <p>Enter name of your want to share:</p>
+      <textarea name="name-input">name</textarea>
+      <p>Enter your Email:</p>
+      <textarea name="email-input">name</textarea>
+      <p>Type of that:</p>
+      <textarea name="type-input">type</textarea>
+      <p>Enter your comment:</p>
```

Comment:
Since Matt is working on the html stuff, it may be worth checking with him to make sure that this file is needed.

Reply:
I was planning on creating a separate page for account management/creating more listings of items/resources. This seems fine considering this is on a separate .html file. However I am unsure on as to what the "getHistory()" functions does on load